google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java

```diff
@@ -228,8 +244,14 @@ static ImmutableList<ListAssetsResult> listAssetsWithQueryMarks(
   static ImmutableList<ListFindingsResult> listFindingsWithQueryMarks(SourceName sourceName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
       // Start setting up a request for to list all findings filtered by a specific security mark.
-      // SourceName sourceName = SourceName.of(/*organization=*/"123234324",/*source=*/
-      // "423432321");
+      // SourceName must be in one of the following formats:
```

Comment:
Same issue as before, where it seems like the list is showing the format of the parent specification, not the SourceName specification.

Reply:
Corrected it. Thank for the suggestion.